prelude/src/stdlib/other/Chars.sk

```diff
@@ -135,6 +135,19 @@ fun octDigitsToInt(digits: String): Int {
   digits.foldl((current, digit) -> current * 8 + hexDigitToInt(digit), 0)
 }
 
+fun intToOctDigits(n: Int): String {
+  digits = mutable Vector[];
+  while (n > 0) {
+    digits.push(hexDigitChars[n.and(0x8)]);
+    !n = n.ushr(8)
```

Comment:
Did you mean
```
    digits.push(hexDigitChars[n.and(7)]);
    !n = n.ushr(3)
```

Reply:
Hum yes. I hastily rewrote some code that was initially using modulo/div. This function is only used (so far) to serialize the file mode in `arparser`, which we do not inspect so it went unnoticed. Thanks!

Reply:
Fixed in #23.